packages/blocks/src/render-blocks.js

```diff
@@ -62,14 +62,34 @@ const parseNodes = ( nodes ) =>
 			name: toLower( node.tagName ),
 			children: parseNodes( node.childNodes ),
 			props: {
-				...attributes,
+				...parseTagProps( toLower( node.tagName ), attributes ),
 				className,
 				style: parseStyles( style ),
 				key: uniqueId( KEY_PREFIX ),
 			},
 		};
 	} );
 
+/**
+ * Parses DOM nodes into an array of objects to be used to create React elements.
+ *
+ * @param  {string} tagName
+ * @param {Object}  props object.
+ * @return {Object}  Props object.
+ */
+const parseTagProps = ( tagName, props ) => {
```

Comment:
I'm actually surprised this works with an arrow function defined below where it's first used.  
Let's maybe still put it on top to keep the dependency flow.